system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.hpp

```diff
@@ -57,6 +57,46 @@ class PeriodicMeasurementNode : public system_metrics_collector::Collector,
    */
   std::string GetStatusString() const override;
 
+  /**
+   * Implementation of the on_activate transition for this LifecycleNode. This calls
```

Comment:
Starts the node...

https://google.github.io/styleguide/cppguide.html#Function_Comments

See the expected style for docstrings in the link.